src/parser_urdf.cc

```diff
@@ -1908,12 +1910,19 @@ void InsertSDFExtensionCollision(tinyxml2::XMLElement *_elem,
       }
     }
   }
+  // If we didn't find the link, emit a warning
+  if (!link_found) {
+    sdfwarn << "<collision> tag with reference[" << _linkName << "] does not exist"
```

Comment:
Should this be `<link>`?
```suggestion
    sdfwarn << "<link> tag with reference[" << _linkName << "] does not exist"
```